src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+
+  for (const key in robot) {
+    if (`${robot[key]}` in inverse) {
+      return null;
+    } else {
+      inverse[`${robot[key]}`] = key;
+    }
```

Comment:
try to make it work without `else` - [checklist ](https://github.com/mate-academy/js_inverse_robot/blob/master/checklist.md)point 4